gammapy/utils/random/tests/test_inverse_cdf.py

```diff
@@ -0,0 +1,37 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from ..inverse_cdf import InverseCDFSampler
+import numpy as np
+from numpy.testing import assert_allclose
+import scipy.optimize
+import scipy.stats as stats
+
+
+def uniform_dist(n_elem):
+    return np.ones(n_elem)
+
+def gauss_dist(x, mu=0, sigma=0.1):
+    return stats.norm.pdf(x,mu,sigma)
+
+def gaus(x, k, sig, mu):
+    return k * np.exp((-(x-mu)**2)/(2*sig**2))   # 1 gauss
+
+def test_uniform_dist_sampling():
+    for i in np.arange(1,100):
+        idx = InverseCDFSampler(uniform_dist(n_elem=100000), random_state=0)
+        sampled_elem = idx.sample(1000)
+        hist = np.histogram(sampled_elem[0],bins=100)
+        assert_allclose((np.mean(hist[0]) * 100./1000.), 1.0, atol=1e-4)
```

Comment:
Maybe add the assert on the standard deviation i.e. call `np.stddev`?

Maybe it would make the test a bit clearer if you convert the sampled indices into coordinates, like so:

```python3
def uniform(x, a=-1, b=1):
    return np.select([x < a, x > b], [0, 0], 1)

x = np.linspace(-10, 10, 1000)
sampler = InverseCDFSampler(pdf=uniform(x))

idx = sampler.sample(int(1e6))

x_sampled = np.interp(idx, np.arange(1000), x)

np.mean(x_sampled)
np.std(x_sampled)
```

Reply:
Done.